lib/Backend/NativeCodeGenerator.cpp

```diff
@@ -3673,7 +3673,7 @@ JITManager::HandleServerCallResult(HRESULT hr, RemoteCallType callType)
 
     // we should not have RPC failure if JIT process is still around
     // since this is going to be a failfast, lets wait a bit in case server is in process of terminating
-    if (WaitForSingleObject(GetJITManager()->GetServerHandle(), 250) != WAIT_OBJECT_0)
+    if (WaitForSingleObject(GetJITManager()->GetServerHandle(), 1000) != WAIT_OBJECT_0)
```

Comment:
We are seeing it pretty rarely, but still hitting sometimes. I guess we can make it even higher, though. I'll bump up to 3 seconds and put behind #define.